modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/NavigationMenuResourceImpl.java

```diff
@@ -713,8 +713,7 @@ private void _updateNavigationMenuItems(
 						siteNavigationMenuItem.getSiteNavigationMenuItemId(),
 						siteId, siteNavigationMenuId);
 
-					siteNavigationMenuItems.remove(
-						existingSiteNavigationMenuItem);
+					siteNavigationMenuItems.remove(siteNavigationMenuItem);
```

Comment:
Hi @dantewang, before SF, here the logic was to delete the original object, now it seems to remove the updated object. I'm not sure whether it's ok or not.

Reply:
@JanisZhang Yeah, it should be another object. Updated, thanks!

Reply:
Thank you!